.github/workflows/test.yml

```diff
@@ -9,6 +9,8 @@ on:
       - opened
       - synchronize
       - reopened
+  schedule:
+    - cron: "0 0 * * 5" # JST 9:00 (Sat)
```

Comment:
**[Consultation]** This setting is sloppy :tired_face:
Do you have any good idea?

Reply:
What is this "sloppy"? It looks like no problem... Could you explain how is this "sloppy"? 👀

Reply:
I want to run a scheduled build once a week.
But I can't think of a good schedule. (e.g. week, time)

Reply:
Hmm... ☺️